puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -24,6 +24,15 @@ bundle install --without=development --jobs=5 --retry=5
 # we need to install node modules for integration tests (which only run on postgresql)
 if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then
   npm install npm@'<6.0.0' # first upgrade to newer npm
+
+  # Test asset precompile
+  NODE_ENV=production $APP_ROOT/node_modules/.bin/npm install
+  bundle exec rake assets:precompile RAILS_ENV=production
+  bundle exec rake webpack:compile RAILS_ENV=production
```

Comment:
We already run this step without the production option in the jenkins:integration task, would it make sense to remove it from there and just run once, having jenkins run the tests on the production version of the bundle?

Reply:
I think you still want that in the rake task so that the rask task is independently runnable (e.g. locally) ?